platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/Feature.kt

```diff
@@ -36,9 +37,16 @@ class Feature(
             }
         }
 
-    fun findUrl(target: String): String? = when (target) {
-        in manifest.links?.keys ?: listOf() -> manifest.links?.get(target)
-        in manifest.links?.values ?: listOf() -> target
-        else -> null
+    fun findUrl(target: String): String? {
```

Comment:
I'd say this link lookup logic should also move into platform core, but we can do that in a followup.

Reply:
I did update it to match the iOS code:
- If `target` matches the link key then return the translated url.
- If `target` matches translated url of any link, then return the target.
- If no match, return nil.

And yeah, this code could be in core, so it is shared and consistent between platforms. But this will be a followup.